br/pkg/utils/backoff.go

```diff
@@ -53,6 +53,10 @@ const (
 	recoveryMaxAttempts  = 16
 	recoveryDelayTime    = 30 * time.Second
 	recoveryMaxDelayTime = 4 * time.Minute
+
+	rawClientMaxAttempts  = 5
```

Comment:
is 5 times(12~13s) retry enough for the special workload?

Can we make it longer?

Reply:
it's mainly for retrying on removing the idle connection and create a new connection, I think it should be done within seconds, setting total retry to be >10s should be safe.